src/wp-includes/blocks.php

```diff
@@ -114,12 +114,13 @@ function register_block_script_handle( $metadata, $field_name, $index = 0 ) {
 			substr_replace( $script_path, '.asset.php', - strlen( '.js' ) )
 		)
 	);
-	if ( ! file_exists( $script_asset_path ) ) {
+	if ( false === $script_asset_path ) {
 		_doing_it_wrong(
 			__FUNCTION__,
 			sprintf(
 				/* translators: 1: Field name, 2: Block name. */
```

Comment:
The translator comment needs to be updated to correctly describe all three placeholders.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/2313/commits/9a48f686c9be64b80bd32649df53ba049e90e171 ✅